client/src/config.js

```diff
@@ -0,0 +1,2 @@
+const GoogleLoginUrl = 'http://localhost:8000/auth/google';
+export default GoogleLoginUrl;
```

Comment:
is this will be the only config? if not then please export const instead of default

Reply:
Right now this is the only config for client-side.

Reply:
But you are using localhost here, I think host will change for different envs.